src/styles/style.css

```diff
@@ -1,39 +1,76 @@
 .veriff-container {
   margin-top: 80px;
-  width: 125px;
+  width: 256px;
 }
 
 .veriff-label {
-  display: inline-block;
-  max-width: 100%;
-  font-size: 12px;
-  color: rgba(0, 0, 0, 0.5);
-  font-weight: normal;
-  margin-bottom: 10px;
+  display: none;
+  margin-bottom: 8px;
+  color: #505366;
+  font-size: 14px;
+  font-weight: 400;
+  overflow: hidden;
+  text-overflow: ellipsis;
 }
 
 .veriff-text {
-    border: 0;
-    outline: 0;
-    background: transparent;
-    border-bottom: 2px solid rgba(0, 0, 0, 0.25);
-    width: 100%;
-    font-size: 14px;
-    border-radius: 0;
-    -webkit-appearance: none;
-    color: black;
-    margin-bottom: 5px;
+  width: 100%;
+  height: 40px;
+  padding: 8px 16px;
+  margin: 0 0 16px;
+  border-radius: 4px;
+  border: 1px solid #e6ebf4;
+  appearance: none;
+  font-size: 16px;
+  background-color: #fafafc;
+  color: #222328;
+  transition-duration: 0.4s;
+  transition-timing-function: ease-in-out;
+  transition-property: background-color, color, border-color, box-shadow, transform;
+}
+
+.veriff-text::placeholder {
+  color: #bdc0cf;
+}
+
+.veriff-text:hover {
+  border-color: #bdc0cf;
+}
+
+.veriff-text:focus {
+  background-color: white;
+  box-shadow: 0 2px 4px 0 rgba(34, 35, 40, 0.08);
+  transform: translateY(-2px);
 }
 
 .veriff-submit {
-  display: inline-block;
+  width: 100%;
+  background-color: #004e5f;
+  color: white;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 40px;
+  letter-spacing: 0.4px;
+
+  display: inline-flex;
```

Comment:
we are targeting ES5 but using flexbox. are there ES5 browsers with flex support?